<ui version="4.0" >
 <class>ThesaurusUi</class>
 <widget class="QDialog" name="ThesaurusUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QTreeWidget" name="meaningsTV" />
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="entryLA" >
       <property name="toolTip" >
        <string>Index entry</string>
       </property>
       <property name="text" >
        <string>&amp;Keyword:</string>
       </property>
       <property name="buddy" >
        <cstring>entryED</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="entryED" >
       <property name="toolTip" >
        <string>Entry</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="selectionLA" >
       <property name="toolTip" >
        <string>The selected entry</string>
       </property>
       <property name="text" >
        <string>&amp;Selection:</string>
       </property>
       <property name="buddy" >
        <cstring>replaceED</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replaceED" >
       <property name="toolTip" >
        <string>The selected entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replacePB" >
       <property name="toolTip" >
        <string>Replace the entry with the selection</string>
       </property>
       <property name="text" >
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closePB" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <tabstops>
  <tabstop>entryED</tabstop>
  <tabstop>meaningsTV</tabstop>
  <tabstop>replaceED</tabstop>
  <tabstop>replacePB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
